lib/runCode.js

```diff
@@ -193,8 +193,7 @@ module.exports = function (opts, cb) {
         // run the opcode
         var result = opFn.apply(null, args)
       } catch (e) {
-        runState.vmError = e.error
```

Comment:
this is where `RevertOpcodeCreate` breaks, using `new VmError(e)` or `new VmError(e.error)` doesn't work, maybe leave the runState error here for e is not in any of the categorized exceptions?

Reply:
I'm not even sure if `vmError` is used anymore after these changes.  Maybe we can remove it.

Reply:
`vmError` is used in the `result` output object by users of the VM (such as Remix)

Reply:
Ah okay.  Best to leave it in then :D

Reply:
Sorry, double checked remix and it uses `exceptionError`. Though there might be some other user which uses `vmError`, so I should add that back it properly